src/components/logIn/FormLogIn.jsx

```diff
@@ -0,0 +1,175 @@
+import React, { useContext, useState } from "react";
+import VideoContext from "../../providers/VideoContext";
+import { Route, Redirect } from "react-router-dom";
+import * as Style from "../styles/styleSheet.js";
+import users from "../../mockUsers/users";
+import youtubeApi from "../../utils/Youtube";
+
+const FormLogIn = () => {
+  const [errorUser, setErrorUser] = useState("");
+  const [errorUserNotFound, setErrorUserNotFound] = useState("");
+  const [user, setUser] = useState("");
+  const [password, setPassword] = useState("");
+  const [errorPassword, setErrorPassword] = useState("");
+  const { userSession, setUserSession } = useContext(VideoContext);
+  const { changinggSetChangingg } = useContext(VideoContext);
+  const { setFavoritesFlag } = useContext(VideoContext);
+  const { setVideoMetaInfo } = useContext(VideoContext);
+
+  var userMap = new Map();
+
+  const onChange = (e) => {
+    setUser(e.target.value);
+    //setUserSessionLocal(e.target.value);
+  };
+  const onChange2 = (e) => {
+    setPassword(e.target.value);
+    //setUserPassLocal(e.target.value);
+  };
+
+  const validate = () => {
+    let isValid = true;
+    if (!user) {
+      setErrorUser("User required");
+      isValid = false;
+    } else if (user.length < 3) {
+      setErrorUser("User length higher than 3");
+      isValid = false;
+    } else {
+      setErrorUser("");
+    }
+    if (!password) {
+      setErrorPassword("Pass required");
+      isValid = false;
+    } else {
+      setErrorPassword("");
+    }
+
+    return isValid;
+  };
+
+  const validateUser = () => {
+    let isValid = true;
+    if (validate()) {
+      if (userMap.get(user)) {
+        if (userMap.get(user).pass !== password) {
+          isValid = false;
+          setErrorUserNotFound("Wrong Pass");
+        }
+      } else {
+        isValid = false;
+        setErrorUserNotFound("User Not Found");
+      }
+    } else {
+      isValid = false;
+    }
+    return isValid;
+  };
+
+  const getmapUsers = () => {
+    let localusers = { ...users };
+
+    for (let i in localusers) {
+      userMap.set(localusers[i].user, localusers[i]);
+    }
+  };
+
+  const videoTendencies = async () => {
+    console.log("do request with params FORMLOIn");
```

Comment:
For debuggin purposes, console.logs are ok, just make sure you leave them out of your commits and PRs.